app/models/pg_search/document.rb

```diff
@@ -8,6 +8,13 @@ class PgSearch::Document < ::ActiveRecord::Base
   scope :featureable, -> { where(searchable_type: ['Journey', 'App']) }
   scope :featured, -> { featureable.where(featured: true) }
 
+  def self.reposition_all
+    Document.featured.each_with_index(1) do |document, index|
+      document.position = index
+      document.save
```

Comment:
Asi staci `document.update!(position: index)` a netreba potom save. Treba ! metodu lebo nech to hodi exception ked to spadne.